src/main/java/com/bootcamp/demo/service/ScooterService.java

```diff
@@ -45,8 +46,15 @@ public String deleteScooter(String documentName) {
         return this.scooterRepository.readScooters().stream().filter(s -> s.getState().equals(State.OUT_OF_USE)).collect(Collectors.toList());
     }
 
-
     public Scooter getScooterByName(String scooterName) {
         return this.scooterRepository.getScooterByName(scooterName);
     }
+
+    public List<Long> returnNumberStates() throws ExecutionException, InterruptedException {
+        long inUse = this.scooterRepository.readScooters().stream().filter(s -> s.getState().equals(State.IN_USE)).count();
+        long inService = this.scooterRepository.readScooters().stream().filter(s -> s.getState().equals(State.IN_SERVICE)).count();
+        long broken = this.scooterRepository.readScooters().stream().filter(s -> s.getState().equals(State.BROKEN)).count();
+        long outOfUse = this.scooterRepository.readScooters().stream().filter(s -> s.getState().equals(State.OUT_OF_USE)).count();
+        return List.of(inUse, inService, broken, outOfUse);
```

Comment:
You return here in the order inUse, inService, broken and outOfUse, but in the page the order is inUse, outOfUse, broken and inService. The values for inService and outOfUse are swapped.

Reply:
I'll change it now